fga-eps-mds/2020.1-stay-safe-docs
# Update architecture document

## Issue 
Resolves #99


## Description
Update the architecture document with new changes to the data model, remove crawler for population data from diagrams and texts, update front-end modules and other minor updates.

## Checklist:



## How to validate?
- Check that the population data crawler has been removed from the diagrams and text and added a reference to the static file `pupulation.json`(diagrams 5.2 and 5.2.1)
- Check that the population json has been updated. (topic 6.2)
- Check that the favorite location table has been added to user-service modeling (diagrams 6.3.1 and 6.3.2)
- Check that the evaluation table has been updated in the `detalhesAvaliacao` field  (diagrams 6.3.1 and 6.3.2)
- Check that the user table has been updated with the field `mostrarNotificacoes` (diagrams 6.3.1 and 6.3.2)
- Check that the front-end modules have been updated with the `@types` and `hooks` modules (topic and diagram 5.3)
- Check that the types of crimes and events that have been updated with `Pedestres` (topic 6.1.1 and 6.3)
- Check that the data sources are correct (topics 6.2 and 6.3.3)
